pages/users/[id].tsx

```diff
@@ -170,6 +153,7 @@ export default function User({ loginContext }: Props) {
 
   const avatarColor = graffitiToColor($user.graffiti)
   const ordinalRank = numberToOrdinal($user.rank)
+  const signupDate = new Date(2021, 11, 1)
```

Comment:
Is this intended to be the testnet start date?

Reply:
It is!

Reply:
Ah, I think it started on Dec. 1 rather than Nov. 1, but also this should be named something like `startDate` instead

Reply:
I'd be happy to rename the variable -- the `Date` constructs are problematic, but that is in fact December 1, 2021